src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherDocumentPreRules.java

```diff
@@ -59,76 +57,39 @@ protected CuDisbursementVoucherTaxService getCuDisbursementVoucherTaxService(){
-        final String disbVchrPaymentMethodCode = dvDocument.getDisbVchrPaymentMethodCode();
-        final boolean isWireTransfer = StringUtils.equals(
-                KFSConstants.PaymentSourceConstants.PAYMENT_METHOD_WIRE,
-                disbVchrPaymentMethodCode);
-        if (disbVchrPaymentMethodCode != null && !isWireTransfer && hasWireTransferValues(dvWireTransfer)) {
-            String questionText = getConfigurationService().getPropertyValueAsString(
-                    CUKFSKeyConstants.QUESTION_CLEAR_UNNEEDED_WIRE_TAB);
-            
-            final boolean clearTab = super.askOrAnalyzeYesNoQuestion(
-                    KFSConstants.DisbursementVoucherDocumentConstants.CLEAR_WIRE_TRANSFER_TAB_QUESTION_ID,
-                    questionText);
-            if (clearTab) {
-                // NOTE: Can't replace with new instance because Foreign Draft uses same object
-                clearWireTransferValues(dvWireTransfer);
-            } else {
-                // return to document if the user doesn't want to clear the Wire Transfer tab
-                super.event.setActionForwardName(KFSConstants.MAPPING_BASIC);
-                tabStatesOK = false;
-            }
-        }
-
-        return tabStatesOK;
-    }
-
     @Override
-    protected boolean hasWireTransferValues(final PaymentSourceWireTransfer dvWireTransfer) {
-        boolean hasValues = super.hasWireTransferValues(dvWireTransfer);
-        final DisbursementVoucherWireTransferExtendedAttribute wireExtension =
-                (DisbursementVoucherWireTransferExtendedAttribute) dvWireTransfer.getExtension();
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrBankStreetAddress());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrBankProvince());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrBankSWIFTCode());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrBankIBAN());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrSortOrTransitCode());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrCorrespondentBankName());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrCorrespondentBankAddress());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrCorrespondentBankRoutingNumber());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrCorrespondentBankAccountNumber());
-        hasValues |= StringUtils.isNotBlank(wireExtension.getDisbVchrCorrespondentBankSwiftCode());
+    protected boolean hasWireTransferValues(final PaymentSourceWireTransfer wireTransfer) {
+        boolean hasValues = super.hasWireTransferValues(wireTransfer);
+        final PaymentSourceWireTransferExtendedAttribute wireExtension =
+                (PaymentSourceWireTransferExtendedAttribute) wireTransfer.getExtension();
+        hasValues |= StringUtils.isNotBlank(wireExtension.getBankStreetAddress());
```

Comment:
If you do end up creating a custom `PaymentSourcePreRulesService` implementation, it would probably be best to move this extended attribute handling into the equivalent method on that service.